assets/dev/scss/frontend/widgets/image.scss

```diff
@@ -4,27 +4,16 @@
 .elementor-widget-image {
 	text-align: center;
 
-	.elementor-image {
+	a {
```

Comment:
This softer selector might allow other selectors to override it. Could you keep it like before?
```
> a,
figure > a
```